discoveryengine/answer_query_sample.py

```diff
@@ -20,6 +20,10 @@
 from google.cloud import discoveryengine_v1 as discoveryengine
 
 # TODO(developer): Uncomment these variables before running the sample.
+# The `answer_generation_spec` below includes all available answer phase options.
```

Comment:
Suggestion: Can you move the #TODO comment to be above project_id, location variables definition ?